lib/rubocop-rspec_rails.rb

```diff
@@ -16,3 +16,22 @@
 
 project_root = File.join(__dir__, '..')
 RuboCop::ConfigLoader.inject_defaults!(project_root)
+
+# FIXME: This is a workaround for the following issue:
+# https://github.com/rubocop/rubocop-rspec_rails/issues/8
+module RuboCop
+  module Cop
+    class AmbiguousCopName # rubocop:disable Style/Documentation
+      prepend(Module.new do
+        def qualified_cop_name(name, path, warn: true)
+          if name == 'RSpec/Rails/HttpStatus'
+            badge = Badge.parse(name)
+            return resolve_badge(badge, qualify_badge(badge).first, path)
+          end
+
+          super
```

Comment:
In cases as this, I prefer having the `super` call in the guard clause. I find that way it’s easier to see when the patch applies and when it doesn’t.

```suggestion
          return super unless name == 'RSpec/Rails/HttpStatus'

          badge = Badge.parse(name)
          resolve_badge(badge, qualify_badge(badge).first, path)
```